### PR TITLE
CT-666 test deduplication behavior of typed tables

### PR DESCRIPTION
**Pull request by zajca**
Jira: CT-666
KBC: https://github.com/keboola/connection/pull/4324

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
dalo mi to zabrat :D bud si to pofixuj nebo zgroupni ty moje navrhy

File: tests/Backend/Snowflake/ImportTypedTableTest.php [hunk 1]
```diff
@@ -22,6 +24,108 @@ public function setUp(): void
...
+        }
+        $this->assertNotNull($firstLoadZeroRow);
+
+        // import same data second time incrementally
+        // file
+        // - is missing row id 0
+        // - has new value for id 26
+        // - has same value for id 24
+        // - has new ids 1,11,25
+        $csvFile = new CsvFile(__DIR__ . '/../../_data/languages.more-rows_not_all_old.csv');
```
Comment (jirkasemmler) [inline tests/Backend/Snowflake/ImportTypedTableTest.php:83]: nechces ten soubor prejmenovat? na neco jako `languages.more-rows_without_zero.csv`. Tech more-rows je tam vic a at je to zrejme co je ucel toho souboru. Jinak to vypada jako nejakej backup

File: tests/Backend/Snowflake/ImportTypedTableTest.php [hunk 2]
```diff
@@ -22,6 +24,108 @@ public function setUp(): void
...
+        $secondLoadZeroRow = null;
+        $secondLoadOtherRows = [];
+        foreach ($secondLoadData as $row) {
+            if ($row['id'] === '0') {
+                $secondLoadZeroRow = $row;
+                continue;
+            }
+            $secondLoadOtherRows[] = $row;
+        }
+        $this->assertNotNull($secondLoadZeroRow);
```
Comment (jirkasemmler) [inline tests/Backend/Snowflake/ImportTypedTableTest.php:115]: ❤️

File: tests/Backend/Snowflake/ImportTypedTableTest.php (hunk 1 above)
Comment (zajca) [reply tests/Backend/Snowflake/ImportTypedTableTest.php]: cajk done

**Pull request merged**